vyper/warnings.py

```diff
@@ -0,0 +1,3 @@
+# TODO: Create VyperWarning class similarly to what is being done with exceptinos?
+class ExceedsContractSizeLimitWarning(Warning):
```

Comment:
Seems perhaps too long of a name?
```suggestion
class ContractSizeLimitWarning(Warning):
```

Reply:
clarity - elegance tradeoff